tests/image-magick/read.spec.ts

```diff
@@ -46,8 +46,8 @@ describe('ImageMagick#read', () => {
                 await bogusAsyncMethod();
             });
         })
-        .rejects
-        .toThrowError('ReadPNGImage');
+            .rejects
```

Comment:
This also looks indented too much?

Reply:
I'm going to fix it.

Reply:
commit ac5a0b4d78665beb494217032bd8a71722dfe5c0.